# Copyright (C) 2001-2018, Python Software Foundation
# For licence information, see README file.
#
msgid ""
msgstr ""
"Project-Id-Version: Python 3\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-07-23 14:38+0200\n"
"PO-Revision-Date: 2023-06-11 22:40+0200\n"
"Last-Translator: Christophe Nanteuil <christophe.nanteuil@gmail.com>\n"
"Language-Team: FRENCH <traductions@lists.afpy.org>\n"
"Language: fr\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Poedit 3.2.2\n"

#: library/asyncio-exceptions.rst:8
msgid "Exceptions"
msgstr "Exceptions"

#: library/asyncio-exceptions.rst:10
msgid "**Source code:** :source:`Lib/asyncio/exceptions.py`"
msgstr "**Code source :** :source:`Lib/asyncio/exceptions.py`"

#: library/asyncio-exceptions.rst:16
msgid ""
"A deprecated alias of :exc:`TimeoutError`, raised when the operation has "
"exceeded the given deadline."
msgstr ""
"Alias obsolète de :exc:`TimeoutError`, levée lorsque l'opération a dépassé "
"le délai donné."

# suit un :
#: library/asyncio-exceptions.rst:21
msgid "This class was made an alias of :exc:`TimeoutError`."
msgstr "cette classe est devenue un alias de :exc:`TimeoutError`."

#: library/asyncio-exceptions.rst:26
msgid "The operation has been cancelled."
msgstr "L'opération a été annulée."

#: library/asyncio-exceptions.rst:28
msgid ""
"This exception can be caught to perform custom operations when asyncio Tasks "
"are cancelled.  In almost all situations the exception must be re-raised."
msgstr ""
"Cette exception peut être interceptée pour effectuer des opérations "
"personnalisées lorsque des tâches asynchrones sont annulées. Dans presque "
"toutes les situations, l'exception doit être relancée."

#: library/asyncio-exceptions.rst:34
#, fuzzy
msgid ""
":exc:`CancelledError` is now a subclass of :class:`BaseException` rather "
"than :class:`Exception`."
msgstr ""
":exc:`CancelledError` est maintenant une sous-classe de :class:"
"`BaseException`."

#: library/asyncio-exceptions.rst:39
msgid "Invalid internal state of :class:`Task` or :class:`Future`."
msgstr "État interne non valide de :class:`Task` ou :class:`Future`."

#: library/asyncio-exceptions.rst:41
msgid ""
"Can be raised in situations like setting a result value for a *Future* "
"object that already has a result value set."
msgstr ""
"Peut être levée dans des situations telles que la définition d'une valeur de "
"résultat pour un objet *Future* qui a déjà une valeur de résultat définie."

#: library/asyncio-exceptions.rst:47
msgid ""
"The \"sendfile\" syscall is not available for the given socket or file type."
msgstr ""
"L'appel système *sendfile* n'est pas disponible pour le connecteur ou le "
"type de fichier donné."

#: library/asyncio-exceptions.rst:50
msgid "A subclass of :exc:`RuntimeError`."
msgstr "Sous-classe de :exc:`RuntimeError`."

#: library/asyncio-exceptions.rst:55
msgid "The requested read operation did not complete fully."
msgstr "L'opération de lecture demandée s'est terminée avant d'être complète."

#: library/asyncio-exceptions.rst:57
msgid "Raised by the :ref:`asyncio stream APIs<asyncio-streams>`."
msgstr "Levée par les API de :ref:`flux asyncio <asyncio-streams>`."

#: library/asyncio-exceptions.rst:59
msgid "This exception is a subclass of :exc:`EOFError`."
msgstr "Cette exception est une sous-classe de :exc:`EOFError`."

#: library/asyncio-exceptions.rst:63
msgid "The total number (:class:`int`) of expected bytes."
msgstr "Nombre total (:class:`int`) d'octets attendus."

#: library/asyncio-exceptions.rst:67
msgid "A string of :class:`bytes` read before the end of stream was reached."
msgstr ""
"Chaîne de :class:`bytes` lue avant que la fin du flux ne soit atteinte."

#: library/asyncio-exceptions.rst:72
msgid "Reached the buffer size limit while looking for a separator."
msgstr ""
"La limite de taille de tampon a été atteinte lors de la recherche d'un "
"séparateur."

#: library/asyncio-exceptions.rst:74
msgid "Raised by the :ref:`asyncio stream APIs <asyncio-streams>`."
msgstr "Levée par les API :ref:`flux asyncio <asyncio-streams>`."

#: library/asyncio-exceptions.rst:78
msgid "The total number of to be consumed bytes."
msgstr "Nombre total d'octets à consommer."
